virttest/utils_params_setup.py

```diff
@@ -0,0 +1,24 @@
+"""
+Virtualization test utility functions.
+"""
+import functools
+
+
+def parameters_replace(func):
+    """Parameters decorator,"""
+
+    @functools.wraps(func)
+    def parameters_decorator(*args, **kwargs):
+        try:
+            mapping = {"NO_EQUAL_STRING": "on"}
+            origin_val = args[0].get_param("serial", "")
```

Comment:
Do we need to check the device type firstly? serial looks a very common option.  Or
we just check the value NO_EQUAL_STRING no matter what key is set.

Reply:
I don't suggest creating such API as the standard interface in the framework since this is just for the qume perspective, but is not available for others.
And this function may be blurry, it is hard to understand what it does, I mean I can not get the purpose of this function. 
And would you like to elaborate on the purpose of this module?
Thanks.
```
"""
Virtualization test utility functions.
"""
```